### ISSUE TITLE
Simplified Chinese translation

### ISSUE THREAD

**Issue by shu307** [emipa606/RelevantStatsInDescription#3]
Sorry, but machine translated chinese sucks, just like my english.
And it seems that `Medical Tend Quality` doesn't work for me ( I only have vanilla beds ).
Is this a vanilla feature? If not, I may have some misunderstanding about it leading to inaccurate translation.

---- edited, added ignored indentation.

```xml
<?xml version="1.0" encoding="utf-8"?>
<LanguageData>
  <!-- Info -->
  <RSID_Undefined>未定义</RSID_Undefined>
  <RSID_Small>小</RSID_Small>
  <RSID_Medium>中</RSID_Medium>
  <RSID_Large>大</RSID_Large>
  <RSID_MaxHP>耐久度上限：{0}</RSID_MaxHP>
  <RSID_MaxAffordance>支持类型：{0}</RSID_MaxAffordance>
  <RSID_Cover>掩体效果：{0}</RSID_Cover>
  <RSID_Beauty>美观度：{0}</RSID_Beauty>
  <RSID_Mass>质量：{0}</RSID_Mass>
  <RSID_Size>尺寸：{0}</RSID_Size>
  <RSID_Wealth>市场价值：{0}</RSID_Wealth>
  <RSID_Cleanliness>清洁度：{0}</RSID_Cleanliness>
  <RSID_Joy>娱乐效率：{0}</RSID_Joy>
  <RSID_JoyKind>娱乐类型：{0}</RSID_JoyKind>
  <RSID_Comfort>舒适度：{0}</RSID_Comfort>
  <RSID_BedRestEffectiveness>休息效率：{0}</RSID_BedRestEffectiveness>
  <RSID_ImmunityGainSpeedFactor>免疫速度系数：{0}</RSID_ImmunityGainSpeedFactor>
  <RSID_SurgerySuccessChanceFactor>手术成功率系数：{0}</RSID_SurgerySuccessChanceFactor>
  <RSID_MedicalTendQuality>治疗质量偏移：{0}</RSID_MedicalTendQuality>
  <RSID_PowerUser>功耗：{0} W</RSID_PowerUser>
  <RSID_PowerProducer>最大输出功率：{0} W</RSID_PowerProducer>
  <RSID_ChemfuelUser>化学燃料消耗：{0}/天</RSID_ChemfuelUser>
  <RSID_GasUser>气体消耗：{0}立方米/天</RSID_GasUser>
  <RSID_GasProducer>最大产气量：{0} 立方米/天</RSID_GasProducer>
  <RSID_AffordanceRequirement>地形需求：{0}</RSID_AffordanceRequirement>
  <RSID_DPM>伤害/分钟：{0}</RSID_DPM>
  <RSID_WorkExact>工作量：{0}</RSID_WorkExact>
  <RSID_WorkRelative>工作量：{0}</RSID_WorkRelative>
  <RSID_TechLevel>科技水平：{0}</RSID_TechLevel>
  <RSID_StorageSpace>最大存储空间：{0}</RSID_StorageSpace>
  <!-- Settings -->
  <RSID_ShowHP.Label>显示可连接结构和门的最大HP</RSID_ShowHP.Label>
  <RSID_ShowHP.Tooltip>显示墙、围栏、门和沙袋等结构的耐久度上限</RSID_ShowHP.Tooltip>
  <RSID_ShowHPForAll.Label>    显示所有掩体的最大HP</RSID_ShowHPForAll.Label>
  <RSID_ShowHPForAll.Tooltip>显示所有可提供掩体效果的建筑物的耐久度上限</RSID_ShowHPForAll.Tooltip>
  <RSID_ShowAffordance.Label>显示支持类型</RSID_ShowAffordance.Label>
  <RSID_ShowAffordance.Tooltip>显示地板和桥梁所提供的最大支持类型</RSID_ShowAffordance.Tooltip>
  <RSID_ShowAffordanceRequirement.Label>显示地形需求</RSID_ShowAffordanceRequirement.Label>
  <RSID_ShowAffordanceRequirement.Tooltip>显示建筑物的最小地形需求</RSID_ShowAffordanceRequirement.Tooltip>
  <RSID_ShowCover.Label>显示掩体效果</RSID_ShowCover.Label>
  <RSID_ShowCover.Tooltip>显示围栏和沙袋等结构所提供的掩体效果</RSID_ShowCover.Tooltip>
  <RSID_ShowBeauty.Label>显示美观度</RSID_ShowBeauty.Label>
  <RSID_ShowBeauty.Tooltip>显示任何建筑物所提供的美观度，正负值都显示出来。</RSID_ShowBeauty.Tooltip>
  <RSID_ShowMass.Label>显示质量</RSID_ShowMass.Label>
  <RSID_ShowMass.Tooltip>显示任何可卸载的建筑物的质量</RSID_ShowMass.Tooltip>
  <RSID_ShowSize.Label>显示尺寸</RSID_ShowSize.Label>
  <RSID_ShowSize.Tooltip>显示建筑物安装时的尺寸</RSID_ShowSize.Tooltip>
  <RSID_ShowWealth.Label>显示市场价值（财富）</RSID_ShowWealth.Label>
  <RSID_ShowWealth.Tooltip>显示任何建筑物所提供的殖民地财富</RSID_ShowWealth.Tooltip>
  <RSID_ShowCleanliness.Label>显示清洁度</RSID_ShowCleanliness.Label>
  <RSID_ShowCleanliness.Tooltip>显示建筑物所提供的清洁度（如果有定义）</RSID_ShowCleanliness.Tooltip>
  <RSID_ShowTechLevel.Label>显示科技水平</RSID_ShowTechLevel.Label>
  <RSID_ShowTechLevel.Tooltip>显示建筑物的科技水平（如果有定义）</RSID_ShowTechLevel.Tooltip>
  <RSID_ShowJoy.Label>显示娱乐效率</RSID_ShowJoy.Label>
  <RSID_ShowJoy.Tooltip>显示一个建筑物的娱乐效率（如果有定义）</RSID_ShowJoy.Tooltip>
  <RSID_ShowJoyKind.Label>显示娱乐类型</RSID_ShowJoyKind.Label>
  <RSID_ShowJoyKind.Tooltip>显示一个建筑物的娱乐类型</RSID_ShowJoyKind.Tooltip>
  <RSID_ShowComfort.Label>显示舒适度</RSID_ShowComfort.Label>
  <RSID_ShowComfort.Tooltip>显示建筑物所提供的舒适度</RSID_ShowComfort.Tooltip>
  <RSID_ShowBedRest.Label>显示休息效率</RSID_ShowBedRest.Label>
  <RSID_ShowBedRest.Tooltip>显示任何床位的休息效率</RSID_ShowBedRest.Tooltip>
  <RSID_ShowStorageSpace.Label>显示最大存储空间</RSID_ShowStorageSpace.Label>
  <RSID_ShowStorageSpace.Tooltip>显示建筑物可存储的最大堆叠数</RSID_ShowStorageSpace.Tooltip>
  <RSID_ShowImmunityGainSpeed.Label>显示免疫速度系数</RSID_ShowImmunityGainSpeed.Label>
  <RSID_ShowImmunityGainSpeed.Tooltip>显示任何默认为医院床位（如果有定义）或动物床位的免疫速度系数</RSID_ShowImmunityGainSpeed.Tooltip>
  <RSID_ShowSurgerySuccessChance.Label>显示手术成功率</RSID_ShowSurgerySuccessChance.Label>
  <RSID_ShowSurgerySuccessChance.Tooltip>显示任何默认为医院床位（如果有定义）或动物床位的手术成功率</RSID_ShowSurgerySuccessChance.Tooltip>
  <RSID_ShowMedicalTendQuality.Label>显示治疗质量偏移</RSID_ShowMedicalTendQuality.Label>
  <RSID_ShowMedicalTendQuality.Tooltip>显示任何默认为医院床位（如果有定义）或动物床位的治疗质量偏移</RSID_ShowMedicalTendQuality.Tooltip>
  <RSID_ShowPowerConsumer.Label>显示功率消耗</RSID_ShowPowerConsumer.Label>
  <RSID_ShowPowerConsumer.Tooltip>将显示任何消耗电力的建筑物的最大耗电量</RSID_ShowPowerConsumer.Tooltip>
  <RSID_ShowPowerProducer.Label>显示输出功率</RSID_ShowPowerProducer.Label>
  <RSID_ShowPowerProducer.Tooltip>将显示任何生产电力的建筑物的最大发电量</RSID_ShowPowerProducer.Tooltip>
  <RSID_ShowVFEGas.Label>    同时适用于天然气生产/消费</RSID_ShowVFEGas.Label>
  <RSID_ShowVFEGas.Tooltip>将显示Vanilla Furniture Expanded - Power中建筑物的气体生产/消费情况：取决于上面的功率设置</RSID_ShowVFEGas.Tooltip>
  <RSID_ShowRimefeller.Label>    同时适用于化学燃料使用</RSID_ShowRimefeller.Label>
  <RSID_ShowRimefeller.Tooltip>将显示Rimefeller中动力装置的化学燃料使用情况：取决于上面的功率设置</RSID_ShowRimefeller.Tooltip>
  <RSID_ShowWorkToBuild.Label>显示工作量</RSID_ShowWorkToBuild.Label>
  <RSID_ShowWorkToBuild.Tooltip>将显示完成建筑所需的工作量</RSID_ShowWorkToBuild.Tooltip>
  <RSID_RelativeWork.Label>    显示相对工作量</RSID_RelativeWork.Label>
  <RSID_RelativeWork.Tooltip>将显示工作量为小/中/大，以获得更多关于工作量的总体概念</RSID_RelativeWork.Tooltip>
  <RSID_RemoveRotateWidget.Label>移除旋转信息窗口</RSID_RemoveRotateWidget.Label>
  <RSID_RemoveRotateWidget.Tooltip>将删除建筑规划菜单上方的旋转信息窗口，如果你认为你现在已经记住了它</RSID_RemoveRotateWidget.Tooltip>
  <RSID_CurrentVersion.Label>已安装的mod版本：v{0}</RSID_CurrentVersion.Label>
</LanguageData>
```

**Comment by emipa606** [comment]
Medical tend quality should be visible on the medical beds

**Comment by shu307** [comment]
It's not an important stat. But I have disabled most of mods and tested, no relevant information displayed.
mod list:
![QQ截图20230220081154](https://user-images.githubusercontent.com/26036149/219984608-3fee1908-be39-4242-945f-3c0713262669.png)
mod settings:
![QQ截图20230220081116](https://user-images.githubusercontent.com/26036149/219984680-8bfe3bc0-e3b4-42ea-a535-6a6130638c95.png)
desc:
![QQ截图20230220080851](https://user-images.githubusercontent.com/26036149/219984781-aac95120-42dd-47c0-9d56-1da66ef0cf43.png)
game version:
![QQ截图20230220082956](https://user-images.githubusercontent.com/26036149/219984959-26f77a33-88c6-4143-8e54-ae684772f12a.png)

**Comment by emipa606** [comment]
I see what the issue is. Should be fixed in 5e35ff4
Thanks for reporting!
